package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "13.2.4",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
```

Comment:
hi didier! I followed the link [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) found on the guidelines. So that's wrong?